<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab_client_3">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>10</y>
      <width>1301</width>
      <height>691</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/imgnull/imagnull.png);
border-image: url(:/back_affiche/affiche.png);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter un Client</string>
     </attribute>
     <widget class="QWidget" name="tabclient" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>1281</width>
        <height>581</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <widget class="QLineEdit" name="NOM_CLIENT">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>230</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent;
font: 20pt &quot;MS Shell Dlg 2&quot;;
    border: none;
font: 9pt &quot;Modern No. 20&quot;;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="PRENOM_CLIENT">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>230</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent;
font: 20pt &quot;MS Shell Dlg 2&quot;;
    border: none;
font: 9pt &quot;Modern No. 20&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="CIN_CLIENT">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>130</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent;
font: 20pt &quot;MS Shell Dlg 2&quot;;
    border: none;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NUMERO_CLIENT">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>330</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent;
font: 20pt &quot;MS Shell Dlg 2&quot;;
    border: none;
font: 9pt &quot;Modern No. 20&quot;;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ADRESSE_CLIENT">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>330</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent;
font: 20pt &quot;MS Shell Dlg 2&quot;;
    border: none;
font: 9pt &quot;Modern No. 20&quot;;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="MAIL_CLIENT">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>440</y>
         <width>531</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent;
font: 20pt &quot;MS Shell Dlg 2&quot;;
    border: none;
font: 9pt &quot;Modern No. 20&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="POSTALE_CLIENT">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>380</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent;
font: 20pt &quot;MS Shell Dlg 2&quot;;
    border: none;
font: 9pt &quot;Modern No. 20&quot;;</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="DATE_CLIENT">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>230</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="VILLE_CLIENT">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>380</y>
         <width>201</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> background-color: transparent;
font: 20pt &quot;MS Shell Dlg 2&quot;;
    border: none;
font: 9pt &quot;Modern No. 20&quot;;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_AJOUTER">
       <property name="geometry">
        <rect>
         <x>1080</x>
         <y>510</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>1201</width>
         <height>561</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../interface ajout.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>label_2</zorder>
      <zorder>NOM_CLIENT</zorder>
      <zorder>PRENOM_CLIENT</zorder>
      <zorder>CIN_CLIENT</zorder>
      <zorder>NUMERO_CLIENT</zorder>
      <zorder>ADRESSE_CLIENT</zorder>
      <zorder>MAIL_CLIENT</zorder>
      <zorder>POSTALE_CLIENT</zorder>
      <zorder>DATE_CLIENT</zorder>
      <zorder>VILLE_CLIENT</zorder>
      <zorder>PB_AJOUTER</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher un Client</string>
     </attribute>
     <widget class="QTableView" name="tab_client">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1271</width>
        <height>691</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/back_affiche/affiche.png);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>supprimer un Client</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 63 8pt &quot;Verdana Pro Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>CIN: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CIN_SUPP">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="BP_SPPRIMER">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 63 8pt &quot;Verdana Pro Semibold&quot;;</string>
      </property>
      <property name="text">
       <string>SUPPRIMER</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1277</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>PRENOM_CLIENT</tabstop>
  <tabstop>DATE_CLIENT</tabstop>
  <tabstop>NUMERO_CLIENT</tabstop>
  <tabstop>ADRESSE_CLIENT</tabstop>
  <tabstop>VILLE_CLIENT</tabstop>
  <tabstop>POSTALE_CLIENT</tabstop>
  <tabstop>MAIL_CLIENT</tabstop>
  <tabstop>PB_AJOUTER</tabstop>
  <tabstop>tab_client_3</tabstop>
  <tabstop>CIN_CLIENT</tabstop>
  <tabstop>NOM_CLIENT</tabstop>
  <tabstop>tab_client</tabstop>
  <tabstop>BP_SPPRIMER</tabstop>
  <tabstop>CIN_SUPP</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
